README.md

```diff
@@ -86,22 +86,45 @@ Here is an example output:
 ```console
 $ sonar ps
 
-2023-07-27T18:29:17+02:00,somehost,8,user,,alacritty,3.7,214932
-2023-07-27T18:29:17+02:00,somehost,8,user,,slack,2.4,1328412
-2023-07-27T18:29:17+02:00,somehost,8,user,,X,0.8,173148
-2023-07-27T18:29:17+02:00,somehost,8,user,,brave,15.5,7085968
-2023-07-27T18:29:17+02:00,somehost,8,user,,.zoom,37.8,1722564
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=.vim-wrapped,cpu%=1.9,cpukib=7228,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=node,cpu%=1.8,cpukib=79332,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=slack,cpu%=0.7,cpukib=591720,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=X,cpu%=1.5,cpukib=224416,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=brave,cpu%=12.1,cpukib=3075300,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=alacritty,cpu%=1.2,cpukib=286196,gpus={},gpu%=0,gpumem%=0,gpukib=0
+v=0.7.0,time=2023-07-29T17:45:37+02:00,host=somehost,cores=12,user=someone,job=0,cmd=sonar,cpu%=9,cpukib=372,gpus={},gpu%=0,gpumem%=0,gpukib=0
 ```
 
 The columns are:
-- time stamp
-- hostname
-- number of cores on this node
-- user
-- Slurm job ID (empty if not applicable)
-- process
-- CPU percentage (as they come out of `ps`)
-- memory used in KiB
+- `v`: version (to make data migrations easier in future)
+- `time`: local time stamp
+- `host`: host name
+- `cores`: number of cores on this node
+- `user` : username owning the process/command
+- `job`: job ID (0 if not applicable)
+- `cmd`: process/command
+- `cpu%`: CPU percentage (as they come out of `ps`)
+- `cpukib`: CPU memory used in KiB
+- `gpus`: GPU devices (more about it below)
```

Comment:
Documentation should specify the range of values; is it zero-based or 1-based?

Reply:
I thought of allowing any string, not only numbers. But that would complicate "none" and "unknown". And for the numbers I thought of allowing whatever base is defined client-side.

Reply:
Re the number base, I sort of feel it's important to settle on one thing, because the analysis tool will not necessarily know what hardware the host has.  Although that is in itself an interesting issue, the current analysis tool does require information about the hardware configuration in some cases, and it would be possible to make better use of that, with care.

I suppose it's possible to leave it open here (ideally, explicitly so) and to account for that in the analysis tool, which does not currently assume a dense set anyway, and to tighten up here later if it becomes an issue.